HappyCamps/src/app/Models/roles.ts

```diff
@@ -1,5 +1,5 @@
 export enum Roles{
-    VOLUNTEER,
-    ORGANIZER,
-    ADMIN
+    VOLUNTEER='VOLUNTEER',
```

Comment:
This is actually pointless, if you want to get the string value of an enum just call Roles[VOLUNTEER] and you get the exact same result

Reply:
This is how i atually get the string. Otherwise wont work

Reply:
Check out this : https://bobbyhadz.com/blog/typescript-convert-enum-to-string

Reply:
ok.